Sources/KSCrashRecordingCore/KSJSONCodec.c

```diff
@@ -702,6 +735,57 @@ static int writeUTF8(unsigned int character, char **dst)
     return KSJSON_ERROR_INVALID_CHARACTER;
 }
 
+static int noOpFloatHandler(const char *const name, const double value, void *const userData)
```

Comment:
Given that these don't return an OK status, maybe they would be better named something like `invalidArgFloatHandler`?